config/clamd.conf

```diff
@@ -1,5 +1,3 @@
 Foreground yes
-DatabaseDirectory /srv/vets-api/clamav/database
-LocalSocket /srv/vets-api/clamav/clamd.ctl
 TCPSocket 3310
-TCPAddr 127.0.0.1
\ No newline at end of file
+TCPAddr clamav
```

Comment:
The hostname of the container. If we name the pods the same, this might work unchanged in k8s?